Code-4-Community/speak-for-the-trees-frontend
# Add planting instructions to empty sites

## Why

[ClickUp Ticket](https://app.clickup.com/t/1dk44ef)


Add planting instructions to empty site tree pages.

## This PR


Display a message with a link to request a tree to be planted at a site at the top of the tree page if the site does not yet have a tree.

## Screenshots

 
![image](https://user-images.githubusercontent.com/33704204/136898399-ec7917c4-c9ae-4c1c-b017-85dad877b55f.png)

## Verification Steps

 
Visual verification that the planting instruction message displayed is correct on Firefox and Chrome. Ran `npm run check` and `npm run test`.
